packages/mp/pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "mp"
-version = "1.9"
+version = "2.0"
```

Let's wait with 2.0 to when the change is really big like adding playbooks.
```suggestion
version = "1.9.1"
```